common/array.h

```diff
@@ -141,6 +145,12 @@ class Array {
 		insert_aux(_storage + idx, array.begin(), array.end());
 	}
 
+	/**
+	 * Inserts element before pos.
+	 */
```

Comment:
`insert_aux` is inserting an element before `pos`. So, this does behave like `std::vector::insert` when it comes to insertion position.